src/data/structs.rs

```diff
@@ -268,16 +274,17 @@ pub struct AcceptResponse {
 }
 
 #[derive(Serialize, Deserialize, Debug, Clone)]
-pub struct BlindedOutpoint {
+pub struct BlindedOrNotOutpoint {
```

Comment:
I think is good we need some functions to the struct, like `is_blinded`, `is_reveled`, etc... 

What do you think?

Reply:
Good idea. And if you have a better name for the struct, it would be good too.